src/@utils/assetConvertor.ts

```diff
@@ -8,17 +7,14 @@ export async function transformAssetToAssetSelection(
   assets: Asset[],
   selectedAlgorithms?: PublisherTrustedAlgorithm[]
 ): Promise<AssetSelectionAsset[]> {
-  const extendedAssets: AssetExtended[] = await getAccessDetailsForAssets(
-    assets
-  )
   const algorithmList: AssetSelectionAsset[] = []
 
-  for (const asset of extendedAssets) {
+  for (const asset of assets) {
     const algoService =
       getServiceByName(asset, 'compute') || getServiceByName(asset, 'access')
 
     if (
-      asset?.accessDetails?.price &&
+      asset?.stats?.price?.value &&
```

Comment:
just double checking: are we using what `transformAssetToAssetSelection()` returns on each asset to then start a compute job with it? If not, then all changes in this file are good. But if not, we would need to fetch actual `accessDetails` for respective assets

Reply:
Yeah, we're not using it for anything in `src/components/Asset/AssetActions/Compute/index.tsx`